pkg/common/common.go

```diff
@@ -71,36 +70,6 @@ func GetNamespaceLabels(namespaceObj *v1.Namespace, logger logr.Logger) map[stri
 	return namespaceUnstructured.GetLabels()
 }
 
-func VariableToJSON(key, value string) []byte {
```

Comment:
Ah, this code... ;-)

Reply:
I have no idea why it was constructing a json string, so much margin for error.

Reply:
Don't want to see this code again !